index.rst

```diff
@@ -164,6 +173,8 @@ DataGraph
 
 A graph in which the nodes are :ref:`DatasetRefs <DatasetRef>` and :ref:`DataUnits <DataUnit>`, and the edges are the relations between them.
 
+[KSK] Don't ``DatasetRef``\ s already contain a tuple of ``DataUnit``\ s?  Maybe an example, very simple ``DataGraph`` would help clarify.
+
```

Comment:
Yes, as I see it, the `DataGraph` class is really just a top level entry point to sets of `DataRefs` and `DataUnits`, which have the necessary attributes themselves to *be* the graph.